frontend/views/containers/proposals/ChangeVotingRules.vue

```diff
@@ -74,6 +74,11 @@ export default ({
       sbp('okTurtles.events/emit', CLOSE_MODAL)
     }
   },
+  mounted () {
+    if (!this.groupShouldPropose && this.ourUsername !== this.groupSettings.groupCreator) {
+      this.$refs.proposal.close()
+    }
```

Comment:
https://github.com/okTurtles/group-income/assets/17641213/06d6183c-517f-4497-96d8-69cd5db3290e

@Silver-IT Apparently, changes here causes this unwanted behavior of the modal opens and then closes immediately. I think hiding the `Change` button is better solution instead of this.

Reply:
Yeah. That's reasonable. Nice catch.